services/explorer/src/pages/AddressInfo/AddressInfo.effects.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { useEffect, useState } from 'react';
+
+const blockData: any[] = Array(20).fill({
+  txHash: '0x00e193a154...',
+  status: 'SUCCESS',
+  timestamp: 'string',
+  pbftBlock: '529133',
+  dagLevel: 'string',
+  dagHash: 'string',
+  value: '0.000000',
+  from: '0xc26f6b31a5...',
+  gasLimit: 'string',
+  gas: '0.000021',
+  gasPrice: 'string',
+  to: '0xc3r17b31a5...',
+  nonce: 153,
+  transactionLink: <span>0x00e193a154...</span>,
```

Comment:
This is wrong. We will not be getting the results like this from Graphql.
Please check TransactionData.effects.ts, inside useEffect and create the objects in a similar way.

Reply:
Ohh I forgot about this one. It was just a mapper object.